tensorflow/tools/api/golden/tensorflow.image.pbtxt

```diff
@@ -173,6 +173,10 @@ tf_module {
     argspec: "args=[\'image_size\', \'bounding_boxes\', \'seed\', \'seed2\', \'min_object_covered\', \'aspect_ratio_range\', \'area_range\', \'max_attempts\', \'use_image_if_no_bounding_boxes\', \'name\'], varargs=None, keywords=None, defaults=[\'None\', \'None\', \'None\', \'None\', \'None\', \'None\', \'None\', \'None\'], "
   }
   member_method {
+    name: "sample_distorted_bounding_box_v2"
+    argspec: "args=[\'image_size\', \'bounding_boxes\', \'min_object_covered\', \'seed\', \'seed2\', \'aspect_ratio_range\', \'area_range\', \'max_attempts\', \'use_image_if_no_bounding_boxes\', \'name\'], varargs=None, keywords=None, defaults=[\'None\', \'None\', \'None\', \'None\', \'None\', \'None\', \'None\'], "
+  }
```

Comment:
Aww, right. We could still hide the op, then switch to V2 in a couple of weeks. @yongtang if you're up for it, you'd need to change https://www.github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/hidden_ops.txt#L159 and write a wrapper like this: http://google3/third_party/tensorflow/python/ops/image_ops_impl.py?l=925